app/code/Magento/Backend/App/Response/Http/FileFactory.php

```diff
@@ -96,17 +96,51 @@ protected function _redirect($path, $arguments = [])
      * @param string $contentType
      * @param int $contentLength    explicit content length, if strlen($content) isn't applicable
      * @return \Magento\Framework\App\ResponseInterface
+     * @throws \Exception
+     *
+     * @deprecated
      */
     public function create(
         $fileName,
         $content,
         $baseDir = DirectoryList::ROOT,
         $contentType = 'application/octet-stream',
         $contentLength = null
+    ) {
+        return $this->createWithDispositionType($fileName, $content, $baseDir, $contentType, $contentLength);
+    }
+
+    /**
+     * Declare headers and content file in response for file download
+     *
+     * @param string $fileName
+     * @param string|array $content set to null to avoid starting output, $contentLength should be set explicitly in
+     * that case
+     * @param string $baseDir
+     * @param string $contentType
+     * @param int $contentLength    explicit content length, if strlen($content) isn't applicable
+     * @param string $dispositionType defines where to show content e.g. attachment or inline
+     * @return \Magento\Framework\App\ResponseInterface
+     * @throws \Exception
+     */
+    public function createWithDispositionType(
```

Comment:
I think it would be better to use the more generic name for the method because you can pass a lot of other params. 
For example: `createWithParams`